vortex_allocator/include/vortex_allocator/allocator_ros.h

```diff
@@ -12,25 +12,24 @@
 class Allocator
 {
 public:
-  explicit Allocator(ros::NodeHandle nh);
-  void callback(const geometry_msgs::Wrench &msg);
+  Allocator(ros::NodeHandle nh);
```

Comment:
Roslint:
`vortex_allocator/include/vortex_allocator/allocator_ros.h:15:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]`